### PR TITLE
[PAY-1736] Implement new ArtistCard on dashboard

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

* Implement ArtistCard
* Refactor and rename USDCCard a little
* Update bg style of stats
* make useCoverPhoto hook match new useProfilePicture pattern

### How Has This Been Tested?

```
npm run web:stage

localhost:3001/dashboard
```

### Screenshots

image

image

### CODE REVIEW

File: packages/web/src/pages/artist-dashboard-page/components/ArtistCard.module.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  border-bottom: 2px solid var(--border-default);
+}
+
+.coverPhoto {
+  height: 100px;
+}
+
+.details {
+  display: flex;
+  padding: 24px;
```
Comment (dharit-tan) [inline packages/web/src/pages/artist-dashboard-page/components/ArtistCard.module.css:22]: nit: use var(--unit-6), also in box-shadow?

File: packages/web/src/pages/artist-dashboard-page/components/ArtistCard.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <DynamicImage
+          wrapperClassName={styles.profilePictureWrapper}
+          image={profilePicture}
+        />
+        <div className={styles.info}>
+          <div className={styles.name}>
+            <Text size='large' strength='default' variant='title'>
+              {name}
+            </Text>
+            <UserBadges userId={userId} badgeSize={14} useSVGTiers />
```
Comment (dharit-tan) [inline packages/web/src/pages/artist-dashboard-page/components/ArtistCard.tsx:38]: nit: is 14 intentional? i think we're supposed to call out any "non-standard" sizes we encounter to design.

File: packages/web/src/pages/artist-dashboard-page/components/USDCCard.tsx [hunk 3]
```diff
@@ -86,13 +86,16 @@ export const USDCTile = ({ balance }: { balance: BNUSDC }) => {
...
-          text={messages.withdraw}
-          // TODO: update leftIcon and wire up withdraw modal https://linear.app/audius/issue/PAY-1754/usdc-withdrawal-flow-ui
-          iconLeft={() => <Icon icon={IconNote} size='medium' />}
-          onClick={() => {}}
-        />
+        <div className={styles.withdrawButton}>
+          <HarmonyButton
+            variant={HarmonyButtonType.SECONDARY}
+            text={messages.withdraw}
+            fullWidth
```
Comment (dharit-tan) [inline packages/web/src/pages/artist-dashboard-page/components/USDCCard.tsx:93]: ty

File: packages/web/src/pages/artist-dashboard-page/components/ArtistCard.tsx (hunk 2 above)
Comment (raymondjacobson) [reply packages/web/src/pages/artist-dashboard-page/components/ArtistCard.tsx:38]: ill call out. badge size is a req param and we use all sorts of wonk values.

**Pull request merged**